docs/wiki/The CI environment - Static validation.md

```diff
@@ -30,6 +30,7 @@ The following activities are run executing the `arm/.global/global.module.tests.
   - at least one `*parameters.json` exists
   - files should be valid JSON
   - contains all required parameters
+  - (if tokens are used) No actual GUID values for tokens used in the environment (i.e. `deploymentSpId`, `subscriptionId`, `managementGroupId`, `tenantId`) are in the parameter files
```

Comment:
Is that only for GUID? I guess it will check for any value stored in a secret named after the token?

Reply:
In theory yes. However, it would not be recommended as e.g. a check for the `namePrefix` could easily be a false positive if (in our case) `carml` is just a substring of something else (e.g. `fancycarmlbaer` and not intended to be treated as a token

Reply:
Will try to rephrase it